### DISCUSSION TITLE
Disable double-space for period

### DISCUSSION THREAD

**Discussion by sethmad** [standardnotes/forum#2703] [General] [answered]
I use the Markdown Basic editor (OS X 3.4.1). Pressing the space bar twice in quick succession inputs a period rather than a double space. That's not helpful when using Markdown, because a double space in Markdown indicates a line break. I find myself repeatedly deleting periods and slowly pressing the space bar twice. I would recommend disabling this feature in the Markdown editor or making it a preference.

**Comment by JaspalSuri** [comment, marked as answer]
Hi Seth,

I'm afraid the only way to disable this would be through System Preferences.

Open System Preferences > click on Keyboard > click on the Text tab > then uncheck the third box from the right that says "Add period with double-space". 

I hope that helps! Let me know if you were asking for something else!

**Comment by sethmad** [comment]
Aha! Apologies, I didn't realize that was an OS feature. Thanks.
